src/ui/store/slice/shareData/shareData.slice.ts

```diff
@@ -38,13 +39,17 @@ export type SetFormItemValuePayload = { id: FormItemId; value: FormItemValue }
 export type initFormItem = FormItem
 export type initFormPayload = initFormItem[]
 
+export type storageServiceId = string
+
 export type ShareDataSlice = {
   shareData: {
     form: Form
+    storageService: O.Option<storageServiceId>
```

Comment:
As we handle an ID, it would have been more accurate to rename it `storageServiceId` and so on for the setter function 😌

Reply:
yes sure!